specification/iotsecurity/resource-manager/Microsoft.IoTSecurity/preview/2021-02-01-preview/locations.json

```diff
@@ -0,0 +1,156 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/LocationList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.IoTSecurity/locations/{iotDefenderLocation}": {
+      "get": {
+        "x-ms-examples": {
+          "Get location": {
+            "$ref": "./examples/Locations/Get.json"
+          }
+        },
+        "tags": [
+          "Locations"
+        ],
+        "operationId": "Locations_Get",
+        "description": "Get a Defender for IoT location associated with the given subscription.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/IotDefenderLocation"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/LocationModel"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "LocationList": {
+      "description": "List of Defender for IoT locations",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List data",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LocationModel"
+          }
+        }
+      }
+    },
+    "LocationModel": {
+      "type": "object",
+      "description": "IoT Defender location",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "IoT Defender location properties",
+          "$ref": "#/definitions/LocationProperties"
+        },
+        "systemData": {
+          "readOnly": true,
+          "type": "object",
+          "description": "Azure Resource Manager metadata containing createdBy and modifiedBy information.",
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
```

Comment:
Not sure having systemData for location makes sense since there is no way for users to create a location..

Reply:
@j5lim Is `systemData` optional for proxy resources?
On a separate PR, we've attempted to create a new proxy resource without `systemData` but received a CI validation error,  
 saying the property must be defined.
Can this error be ignored for proxy resources?

Reply:
The ARM guideline says it should be supported by all resources, but I mean in this particular case it does not make sense to me. Wondering there is a way to suppress the CI validation.. @ruowan do you know if it's possible?

Reply:
@j5lim @ruowan Any decision on this? I'd like to complete this PR (we have some other dependencies on completing it)

Reply:
@liranc You can leave it as it is if it causes a CI failure.

Reply:
@j5lim Can the PR be merged in that case, or are there any other action items left? 🙂

Reply:
@liranc Please work with the pr assignee to get the pr merged. ARM team does not have merge permission.